msvc/SPIRV-Cross.sln

```diff
@@ -1,26 +1,26 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Express 2013 for Windows Desktop
```

Comment:
Why all the changes to MSVC build? Was it accidentally checked in?

Reply:
the commit 'modify visual studio solution to vs2015' is a accidentally checked in, sorry

Reply:
Ok, please rebase and remove the commit in that case rather than revert for commit history cleanliness.